crates/domain/src/models/task.rs

```diff
@@ -11,8 +11,8 @@ pub struct Task {
     pub description: Option<String>,
     pub status: Status,
     pub tags: Option<Vec<String>>,
-    pub created_at: NaiveDateTime,
-    pub updated_at: NaiveDateTime,
+    pub created_at: DateTime<Utc>,
```

Comment:
Let's add `#[serde(rename = "createdAt")]` and `#[serde(rename = "updatedAt")]` to make the json object key camelCase. Same thing applies to line 9.

Reply:
Can I just do `#[serde(rename_all = "camelCase")]`?

Reply:
Ah, I didn't know that is possible. That sounds simpler.